packages/product-react/src/components/Card/Card.scss

```diff
@@ -0,0 +1,15 @@
+.lucia-card--compact {
+  padding: 1rem;
```

Comment:
We can variable-ize these values; Also, we should change the `lucia` name to something else lol

Reply:
We can probably dry these up by nesting `&--compact` and `&--inactive` under `.lucia` (after we change it to some other name :-P)

Reply:
yes agree, but I have no control over his name, maybe it can be changed in another pull request for the project where it's defined?